index.html

```diff
@@ -539,7 +542,7 @@ <h4>XBOX SERIES X CONTROLLER</h4>
           <h2>What Gamers Offer!</h2>
```

Comment:
Don't use padding to make outer space, please fix in all titles

Reply:
Done